frontend/src/Components/Table/TableOptions/TableOptionsModal.js

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { DndProvider } from 'react-dnd';
-import { HTML5Backend } from 'react-dnd-html5-backend';
+import HTML5Backend from 'react-dnd-html5-backend';
```

Comment:
Revert this change, it needs the brackets

Reply:
Incoming commit to fix this